### PR TITLE
extend cutomized recording size and add fb photo viewer case

### PR DESCRIPTION
**Pull request by MikeLien**
1. extract browser and recording size setting from env to exec config
2. add fb photo-viewer case
3. modify browser size related cases to use exec config setting

### CODE REVIEW

**Comment by MikeLien** [comment]
@ShakoHo @askeing @ypwalter r?

File: lib/baseTest.py [hunk 1]
```diff
@@ -195,6 +196,34 @@ def load_configs(self):
...
+            'video-recording-pos-x': self.env.DEFAULT_VIDEO_RECORDING_POS_X,
+            'video-recording-pos-y': self.env.DEFAULT_VIDEO_RECORDING_POS_Y,
+            'video-recording-width': self.env.DEFAULT_VIDEO_RECORDING_WIDTH,
+            'video-recording-height': self.env.DEFAULT_VIDEO_RECORDING_HEIGHT
+        }
+        self.set_configs(self.config_name.EXEC, screen_settings)
+
+    def get_new_recording_size(self):
+        recording_width = min(self.exec_config['browser-width'] + 100, 1920)
+        if platform.system().lower() == "darwin":
```
Comment (ypwalter) [inline lib/baseTest.py]: There is a `self.current_platform_name` for `platform.system()`, and it's lower case by default.

File: tests/regression/facebook/test_chrome_facebook_ail_click_photo_viewer_right_arrow.sikuli/test_chrome_facebook_ail_click_photo_viewer_right_arrow.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import os
+import common
+import facebook
+import shutil
+import browser
+import time
+
+# Disable Sikuli action and info log
+com = common.General()
+com.infolog_enable(0)
```
Comment (ypwalter) [inline tests/regression/facebook/test_chrome_facebook_ail_click_photo_viewer_right_arrow.sikuli/test_chrome_facebook_ail_click_photo_viewer_right_arrow.py]: You can also try to use @askeing 's new `infolog_enable(False)`

File: tests/regression/facebook/test_chrome_facebook_ail_click_photo_viewer_right_arrow.sikuli/test_chrome_facebook_ail_click_photo_viewer_right_arrow.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+chrome = browser.Chrome()
+fb = facebook.facebook()
+
+chrome.clickBar()
+chrome.enterLink(sys.argv[3])
+wait(fb.comment_icons, 15)
```
Comment (ypwalter) [inline tests/regression/facebook/test_chrome_facebook_ail_click_photo_viewer_right_arrow.sikuli/test_chrome_facebook_ail_click_photo_viewer_right_arrow.py]: for `wait(fb.comment_icons)` and `hover(fb.comment_icons.targetOffset(-340, -40)`
It exists in 2 files (this case with 2 different browsers). Do you want to make them functions?

**Pull request merged**